build/ci/create_env_file.py

```diff
@@ -0,0 +1,61 @@
+#!/usr/bin/env python
+
+import os
+
+
+def get_env_var(var):
+    try:
+        return os.environ[var]
+    except:
+        return ""
+
+
+def get_value_from_file(file):
+    try:
+        tf = open(file, "r")
+        return tf.read()
+    except:
+        return ""
+
+
+def write_env_to_file(key, file):
+    var = get_env_var(key)
+    if len(var) > 0:
+        file.write("{}={}\r\n".format(key.upper(), var))
+
+
+with open("environment", "w") as text_file:
+    docker_login = get_value_from_file("docker_login.file")
```

Comment:
This pattern of reading a value from a file, checking for emptiness and then writing it to the environment file under a certain key is repeated several times so it might be cleaner to introduce a function to do so?

Reply:
Fixed